packages/zoltan/src/CMakeLists.txt

```diff
@@ -400,6 +400,7 @@ APPEND_SET(SOURCES
   rcb/inertial3d.c
   rcb/point_assign.c
   rcb/rcb_box.c
+  rcb/rcb_partition_tree.c
```

Comment:
Should add rcb_partition_tree.h to CMakeLists.txt as well, right?
Actually, this question is harder.  Since Zoltan2 uses rcb_partition_tree.h in a *.hpp file, rcb_partition_tree.h will have to be installed with other Zoltan and Zoltan2 header files when one does "make install".  
In that case, rcb_partition_tree.h needs to be named zoltan_rcb_partition_tree.h.  Or maybe we could have a single installed header (e.g., zoltan_partition_tree.h) with prototypes for both RCB and PHG.
Also, both the *.c and *.h files should be added to zoltan/src/Makefile.am as well for Zoltan's autotools build.